### PR TITLE
update gradle/baseline and follow up pr

### PR DESCRIPTION
**Pull request by qinfchen**

### CODE REVIEW

**Comment by qinfchen** [comment]
@jmcampanini CR please.

File: gradle/jacoco.gradle [hunk 1]
```diff
@@ -0,0 +1,44 @@
+apply plugin: "jacoco"
+
+jacoco {
+  toolVersion = "0.7.6.201602180812"
```
Comment (jmcampanini) [inline gradle/jacoco.gradle]: do we have to set a specific version? can we not use gradle's default version?

File: build.gradle [hunk 2]
```diff
@@ -30,7 +30,6 @@ subprojects {
     apply plugin: 'java'
     apply plugin: 'com.palantir.baseline-checkstyle'
     apply plugin: 'com.palantir.baseline-eclipse'
-    apply plugin: 'com.palantir.baseline-findbugs'
```
Comment (jmcampanini) [inline build.gradle:33]: if we remove findbugs, shouldn't we add errorprone?

File: dropwizard-index-page/build.gradle [hunk 3]
```diff
@@ -1,4 +1,4 @@
-apply plugin: 'com.palantir.jacoco-coverage'
+apply from: "${rootDir}/gradle/jacoco.gradle"
```
Comment (jmcampanini) [inline dropwizard-index-page/build.gradle:1]: should this go in the subproject block?

File: dropwizard-index-page/src/main/java/com/palantir/indexpage/IndexPageServlet.java [hunk 4]
```diff
@@ -54,11 +54,11 @@ public IndexPageServlet(String contextPath, String indexPagePath) {
         }
 
         ImmutableMap<String, String> templateContext = ImmutableMap.of(BASE_URL, slashedContextPath);
-        Optional<URL> indexPageResource = tryGetResource(indexPagePath);
-        if (indexPageResource.isPresent()) {
-            indexPage = new ClasspathIndexPage(templateContext, indexPageResource.get());
+        File indexPageFile = new File(indexPagePath);
```
Comment (jmcampanini) [inline dropwizard-index-page/src/main/java/com/palantir/indexpage/IndexPageServlet.java:57]: is this file's changes related to baseline ?

Comment (qinfchen) [reply dropwizard-index-page/src/main/java/com/palantir/indexpage/IndexPageServlet.java:57]: this is to revert the logic change from Henry's pr in order to keep the old behavior

File: dropwizard-index-page/build.gradle (hunk 3 above)
Comment (qinfchen) [reply dropwizard-index-page/build.gradle:1]: no, example project doesn't need it.

File: build.gradle (hunk 2 above)
Comment (qinfchen) [reply build.gradle:33]: prefer separate pr, errorprone uses guava 0.21.

Comment (jmcampanini) [reply build.gradle:33]: sure, but lets make sure we do it. findbugs is removable if you replace it with an equivalent (i.e. error-prone).

uses guava 21 in the gradle classpath, or in the application's classpath? if it's buildscript it should be fine - or is there a plugin that has a conflict?

File: dropwizard-index-page/src/main/java/com/palantir/indexpage/IndexPageServlet.java (hunk 4 above)
Comment (jmcampanini) [reply dropwizard-index-page/src/main/java/com/palantir/indexpage/IndexPageServlet.java:57]: oh. just re-read title of PR :)

if this PR has multiple PRs, might be worth doing the errorprone change in here as well

**Review by jmcampanini — APPROVED** [review]
seems reasonable as long as we get error prone in

File: build.gradle (hunk 2 above)
Comment (qinfchen) [reply build.gradle:33]: yeah, classpath conflict, but pretty easy to fix.

File: dropwizard-index-page/src/main/java/com/palantir/indexpage/IndexPageServlet.java (hunk 4 above)
Comment (qinfchen) [reply dropwizard-index-page/src/main/java/com/palantir/indexpage/IndexPageServlet.java:57]: should've made it into separate PRs :)

**Pull request merged**